nextflow.config

```diff
@@ -10,257 +10,266 @@
+    save_annotations                                        = false
+
+    annotation_prodigal_singlemode                          = false
+    annotation_prodigal_closed                              = false
+    annotation_prodigal_transtable                          = 11
+    annotation_prodigal_forcenonsd                          = false
+
+    annotation_pyrodigal_singlemode                         = false
+    annotation_pyrodigal_closed                             = false
+    annotation_pyrodigal_transtable                         = 11
+    annotation_pyrodigal_forcenonsd                         = false
+
+    annotation_bakta_db                                     = null
+    annotation_bakta_db_downloadtype                        = 'full'
+    annotation_bakta_singlemode                             = false
+    annotation_bakta_mincontiglen                           = 1
+    annotation_bakta_translationtable                       = 11
+    annotation_bakta_gram                                   = '?'
+    annotation_bakta_complete                               = false
+    annotation_bakta_renamecontigheaders                    = false
+    annotation_bakta_compliant                              = false
+    annotation_bakta_trna                                   = false
+    annotation_bakta_tmrna                                  = false
+    annotation_bakta_rrna                                   = false
+    annotation_bakta_ncrna                                  = false
+    annotation_bakta_ncrnaregion                            = false
+    annotation_bakta_crispr                                 = false
+    annotation_bakta_skipcds                                = false
+    annotation_bakta_pseudo                                 = false
+    annotation_bakta_skipsorf                               = false
+    annotation_bakta_gap                                    = false
+    annotation_bakta_ori                                    = false
+    annotation_bakta_activate_plot                          = false
+
+    annotation_prokka_singlemode                            = false
+    annotation_prokka_rawproduct                            = false
+    annotation_prokka_kingdom                               = 'Bacteria'
+    annotation_prokka_gcode                                 = 11
+    annotation_prokka_cdsrnaolap                            = false
+    annotation_prokka_rnammer                               = false
+    annotation_prokka_mincontiglen                          = 1
+    annotation_prokka_evalue                                = 0.000001
+    annotation_prokka_coverage                              = 80
+    annotation_prokka_compliant                             = true
+    annotation_prokka_addgenes                              = false
+    annotation_prokka_retaincontigheaders                   = false
+
+    // Protein_annotation options
+    run_protein_annotation_interproscan                     = false
```

Comment:
For future proofing please make sure to follow the same structure as before:

As this`run_proteinannotation` and then have another parameter `--proteinannotation_tools`, like for tax classiciation

Reply:
Shall Imodify it now or once we add another annotation workflow in the pipeline?

Reply:
Do it now, as otherwise it'll cause breaking changes for the users in the future

Reply:
Done :)